android/build.gradle

```diff
@@ -61,6 +63,10 @@ dependencies {
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.2'
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.5.2'
     implementation 'com.amplitude:analytics-connector:1.0.0'
+    implementation "androidx.core:core-ktx:1.7.0"
+    compileOnly compose.runtime
+    compileOnly compose.ui
```

Comment:
Are they the BOM definitions? Will this only include the indivdual subpackages we need? 

If there is only one or two compose packages required would it be better to include them directly?

Reply:
Here, Compose is handled by the Compose Gradle plugin specified as a classpath. This setup pulls in only the necessary subpackages (like ui, runtime, etc.), so we don't have to specify versions for each one we add now or later.

Since Compose comes from the environment, I'm guessing that setting a specific version might lock the customer to that version. The way it’s set up now, it should work with all Compose versions.